app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,18 @@
+from app.players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(
+            self,
+            nickname: str,
+            musical_instrument: str,
+            bow_level: int) -> None:
+        super().__init__(nickname, musical_instrument)
+        self.bow_level = bow_level
+
+    def get_rating(self) -> int:
+        return self.bow_level * 3
```

Comment:
Ensure that the `get_rating` method accesses the `_bow_level` attribute if you change its name to be protected.